src/momento/config/configuration.py

```diff
@@ -3,12 +3,21 @@
 from abc import ABC, abstractmethod
 from datetime import timedelta
 
+from momento.retry import RetryStrategy
```

Comment:
@malandis does it matter whether we do the import like this vs. the `.` style below?

Reply:
They're equivalent. When the nesting gets deep I prefer absolute imports as following relative imports can get confusing.